File: Going Dutch/Views/MCAllTripsTableViewCell.m
//
//  MCAllTripsTableViewCell.m
//  Going Dutch
//
//  Created by Mark Cornelisse on 18-02-13.
//  Copyright (c) 2013 Mark Cornelisse. All rights reserved.
//

#import "MCAllTripsTableViewCell.h"

@implementation MCAllTripsTableViewCell

@synthesize tripLabel;
@synthesize totalCostLabel;
@synthesize peoplePresentLabel;
@synthesize extraLabel;

- (id)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString *)reuseIdentifier
{
    self = [super initWithStyle:style reuseIdentifier:reuseIdentifier];
    if (self) {
        // Initialization code
    }
    return self;
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated
{
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

@end

File: Going Dutch/MCAllTripsTableViewCell.swift
//
//  MCAllTripsTableViewCell.swift
//  We all pay
//
//  Created by Mark Cornelisse on 11/06/15.
//  Copyright (c) 2015 Mark Cornelisse. All rights reserved.
//

import UIKit

final class MCAllTripsTableViewCell: UITableViewCell {
    @IBOutlet var waitingForXRatesIndicator: UIActivityIndicatorView!
    @IBOutlet var tripLabel: UILabel!
    @IBOutlet var totalCostLabel: UILabel!
    @IBOutlet var peoplePresentLabel: UILabel!
    @IBOutlet var extraLabel: UILabel!
    
    // MARK: UITableViewCell
    
    // MARK: UIView
    
    // MARK: UIResponder
    
    // MARK: NSObject
    
    override func awakeFromNib() {
        if #available(iOS 13.0, *) {
            tripLabel.backgroundColor = .clear
            totalCostLabel.backgroundColor = .clear
            peoplePresentLabel.backgroundColor = .clear
            extraLabel.backgroundColor = .clear
        }
    }
}
